.flake8

```diff
@@ -1,3 +1,12 @@
 [flake8]
-max-line-length = 120
-ignore = E501
+# The following are ignored:
+#   E203 whitespace before ':' (https://github.com/ambv/black#line-breaks--binary-operators)
+#   E501 line too long         (https://github.com/ambv/black#line-length)
+#   W503 line break before binary operator
+ignore = E203, E501, W503
+max-line-length = 88
```

Comment:
Noooo... could we set black to make it at least 100 please? (Personally I'd even vote for keeping 120, but the rationale seems https://github.com/ambv/black#line-length kind-of convinces me...)

Reply:
Ok. I was going with the defaults for black, but 100 is a good compromise.